### PR TITLE
Do not swallow error response

### PR DESCRIPTION
**Pull request by remix7531**
quick fix for [issue #47](https://github.com/bitcoindevkit/rust-esplora-client/issues/47#issuecomment-1627389294)

### CODE REVIEW

**Comment by vladimirfomene** [comment]
@tnull and @notmandatory this is ready for review.

File: .github/workflows/cont_integration.yml [hunk 1]
```diff
@@ -50,7 +50,12 @@ jobs:
       run: rustup update
     - name: pin dependencies
       if: matrix.rust.version == '1.57.0'
-      run: cargo update -p log --precise 0.4.18 && cargo update -p rustls:0.21.2 --precise 0.21.1 && cargo update -p time:0.3.15 --precise 0.3.13
+      run: |
+        cargo update -p log --precise 0.4.18 && 
+        cargo update -p rustls:0.21.5 --precise 0.21.1 && 
+        cargo update -p time:0.3.15 --precise 0.3.13 &&
+        cargo update -p hyper-rustls:0.24.1 --precise 0.24.0 &&
+        cargo update -p tempfile:3.7.0 --precise 3.6.0
```
Comment (notmandatory) [inline .github/workflows/cont_integration.yml:58]: The `&&` aren't needed when put on new lines like this.

```suggestion
        cargo update -p log --precise 0.4.18
        cargo update -p rustls:0.21.5 --precise 0.21.1
        cargo update -p time:0.3.15 --precise 0.3.13
        cargo update -p hyper-rustls:0.24.1 --precise 0.24.0
        cargo update -p tempfile:3.7.0 --precise 3.6.0
```

**Review by notmandatory — APPROVED** [review]
utACK with one suggestion for `cont_integration.yml`. Also be sure to do a final `cargo fmt`.

**Comment by vladimirfomene** [comment]
I'm going to close this, the requested changes have been implemented in #58.

**Pull request closed without merging**